src/com/lingoport/demo/java/ExampleMain.java

```diff
@@ -42,7 +42,8 @@ public static void main(String[] args){
 
     	}
     	
-    	I18nUtils.setLocale(locale);
+    	String unUnusedGuy = " hello world ";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "unUnusedGuy" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9010/coding_rules#rule_key=squid%3AS1481)